.github/workflows/multicoretests.yml

```diff
@@ -0,0 +1,72 @@
+# Run the multicoretests testsuite if PR is labelled with run-multicoretests
+name: Run multicoretests testsuite
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+
+# Restrict the GITHUB_TOKEN
+permissions: {}
+
+# See build.yml
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name == 'pull_request' || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  multicoretests:
+    if: contains(github.event.pull_request.labels.*.name, 'run-multicoretests')
+    runs-on: 'ubuntu-latest'
+    strategy:
+      matrix:
+        include:
+          - profile: dev
+            runparam: b
+          - profile: debug-runtime
+            runparam: b,s=4096
```

Comment:
I'd name this `ocamlrunparam`, I think it's to be read as ocamlrun-param rather than ocaml-runparam as it's literally the parameters given to ocamlrun.